src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -937,20 +940,31 @@ static bool _parseStyleAttr(void* data, const char* key, const char* value)
 
     //Trim the white space
     key = _skipSpace(key, nullptr);
-
     value = _skipSpace(value, nullptr);
 
     sz = strlen(key);
     for (unsigned int i = 0; i < sizeof(styleTags) / sizeof(styleTags[0]); i++) {
         if (styleTags[i].sz - 1 == sz && !strncmp(styleTags[i].tag, key, sz)) {
-            styleTags[i].tagHandler(loader, node, value);
+            if (style) {
+                styleTags[i].tagHandler(loader, node, value);
+                node->style->flags = (SvgStyleFlags)((int)node->style->flags | (int)styleTags[i].flag);
+            } else if (!((int)node->style->flags && (int)styleTags[i].flag)) {
+                styleTags[i].tagHandler(loader, node, value);
+            }
```

Comment:
@JSUYA If you think that it's a good idea I can duplicate this func. 
in the one for style=true we dont need 'if(style)', but the one for style=false still has to check 'if (!((int)node->style->flags && (int)styleTags[i].flag))'. 
I wouldn't duplicate this func, but if you wish, I'll do it. 

I did that for the '_attrParseStops' - I defined '_attrParseStopsStyle' for the case style=true. But in this case it was necessary since other way I'd have to modify the XmlParser.

Reply:
ok I agree to your patch.